.gitignore

```diff
@@ -6,7 +6,7 @@ controlplane.yaml
 join.yaml
 docgen
 talosconfig
-kubeconfig
+/kubeconfig
```

Comment:
The leading slash is needed?

Reply:
yep, as I think intention was to ignore `kubeconfig` at the root (from `osctl kubeconfig`), but in this form it ignores any file/directory named `kubeconfig`, as I added `internal/pkg/kubeconfig`, I need to make sure it's not ignored